modules/apps/questions/questions-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -55,6 +55,8 @@
 				"userId", String.valueOf(themeDisplay.getUserId())
 			).put(
 				"useTopicNamesInURL", questionsConfiguration.useTopicNamesInURL()
+			).put(
```

Comment:
Can you send the value instead that controls if the question is anwered/posted directly? that way we can show approved users the normal message, because those approved users "won't submit the question for publication"

Reply:
Sure. I've thought about it but discarded it because I didn't know if we were capable of checking it and MessageBoardMessage wasn't doing that check. It is true that this way it is misleading seeing this label when you can post directly. I'll do it that way